test/Conversion/TorchToLinalg/flatten.mlir

```diff
@@ -82,3 +82,68 @@ func.func @torch.aten.flatten.using_ints$rank0(%arg0: !torch.vtensor<[],f32>) ->
   %0 = torch.aten.flatten.using_ints %arg0, %int0, %int0 : !torch.vtensor<[],f32>, !torch.int, !torch.int -> !torch.vtensor<[1],f32>
   return %0 : !torch.vtensor<[1],f32>
 }
+
+func.func @torch.aten.view$twotothree(%arg0: !torch.vtensor<[3,2],f32>) -> !torch.vtensor<[2,3],f32> {
+    %0 = builtin.unrealized_conversion_cast %arg0 : !torch.vtensor<[3,2],f32> to tensor<3x2xf32>
+    %int3 = torch.constant.int 3
+    %int2 = torch.constant.int 2
+    %1 = torch.prim.ListConstruct %int2, %int3 : (!torch.int, !torch.int) -> !torch.list<int>
+    %2 = torch_c.to_i64 %int2
+    %3 = torch_c.to_i64 %int3
+    %c0 = arith.constant 0 : index
+    %c3 = arith.constant 3 : index
+    %c1 = arith.constant 1 : index
+    %c2 = arith.constant 2 : index
+    %4 = tensor.cast %0 : tensor<3x2xf32> to tensor<3x2xf32>
+    %5 = tensor.collapse_shape %4 [[0, 1]] : tensor<3x2xf32> into tensor<6xf32>
```

Comment:
why is there collapse/expand in the input to this test?
Also this test doesn't seem to check anything.